tracker/tracker.go

```diff
@@ -152,7 +152,8 @@ func randomString(s string, n int) string {
 	b := make([]byte, n)
 	slen := len(s)
 	for i := 0; i < n; i++ {
-		b[i] = s[rand.Intn(slen)]
+		r := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
It would be best to create the new random source just once, instead of on each loop iteration. Please move this declaration to before the "for" loop.